homeassistant/components/zwave/__init__.py

```diff
@@ -399,11 +411,20 @@ def value_added(node, value):
                 value.enable_poll(polling_intensity)
             else:
                 value.disable_poll()
+            platform = get_platform(component, DOMAIN)
+            device = platform.get_device(
+                node=node, value=value, node_config=node_config, hass=hass)
```

Comment:
Also most platforms don't need all 4 - won't it look better if each one would "declare" only what it needs?